allauth_2fa/views.py

```diff
@@ -84,8 +85,28 @@ def get(self, request, *args, **kwargs):
         self._new_device()
         return super().get(request, *args, **kwargs)
 
+    def get_qr_code_kwargs(self):
+        """Keyword arguments to give to the QR code function.
+
+        This is a separate method so that it can be overridden if needed.
+
+        The fields are:
+        - issuer: The name of the site (e.g. "example.com")
+        - label: The label for the QR code (e.g. "example.com: username")
+        """
```

Comment:
The docstring could use some tweaking, and we can just as well be helpful and paraphrase [the spec for `otpauth` URLs](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).

Let's slap a type annotation on while we're at it.

```suggestion
    def get_qr_code_kwargs(self) -> dict[str, str]:
        """
        Get the configuration for generating a QR code.

        The fields required are:
        - `label`: identifies which account a key is associated with. Contains an account name, preferably prefixed by an issuer name and a colon, e.g. `issuer: account`.
        - `issuer`: indicates the provider or service this account is associated with.
        """
```